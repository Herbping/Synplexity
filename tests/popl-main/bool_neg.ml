type bool =
  | True
  | False

let bool_neg : bool -> bool |>
/\(True -> False, False -> True) = ?
