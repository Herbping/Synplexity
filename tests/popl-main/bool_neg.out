(* Generated Thu Oct  8 20:40:48 2015 *)
fun (b1:bool) ->
  match b1 with
  | True u1 -> False
  | False u1 -> True
